test/platform/github/gh-got-wrapper.spec.js

```diff
@@ -1,13 +1,16 @@
 const get = require('../../../lib/platform/github/gh-got-wrapper');
 const ghGot = require('gh-got');
+const delay = require('delay');
 
 jest.mock('gh-got');
+jest.mock('delay');
 
 describe('platform/gh-got-wrapper', () => {
   const body = ['a', 'b'];
   beforeEach(() => {
     jest.resetAllMocks();
     get.setAppMode(false);
+    delay.mockImplementation(() => Promise.resolve());
```

Comment:
Is there anything peculiar about this file why you need to run it before each test instead of once?